development-docs/systemtests/io.strimzi.systemtest.cruisecontrol.CruiseControlST.md

```diff
@@ -33,6 +33,31 @@
 * [cruise-control](labels/cruise-control.md)
 
 
+## testAutoKafkaRebalanceScaleUpScaleDown
+
+**Description:** Test the Kafka Cruise Control auto-rebalance mechanism during scaling up and down of brokers.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create broker and controller KafkaNodePools. | Both KafkaNodePools are successfully created. |
+| 2. | Create KafkaRebalance templates for scale-up and scale-down operations. | KafkaRebalance templates are created and annotated as configuration templates. |
+| 3. | Deploy Kafka cluster with Cruise Control using the defined templates for auto-rebalance. | Kafka cluster with Cruise Control is deployed, configured with the specified auto-rebalance templates. |
+| 4. | Ensure the Kafka auto-rebalance status is Idle. | Kafka auto-rebalance status is confirmed to be Idle. |
+| 5. | Scale Kafka up to a higher number of brokers. | Kafka brokers are scaled up, and Cruise Control initiates rebalancing in ADD_BROKERS mode. |
+| 6. | Verify that Kafka auto-rebalance status transitions to RebalanceOnScaleUp and then back to Idle. | Auto-rebalance status moves to RebalanceOnScaleUp during scaling and returns to Idle after rebalancing completes. |
+| 7. | Check that topic replicas are moved to the new brokers. | Topic replicas are distributed onto the newly added brokers. |
+| 8. | Change number of replicas of Kafka cluster to initial replicas within KafkaNodePool (i.e., 3 brokers) | KafkaNodePool is set successfully to 3 replicas and auto-rebalance is triggered. |
```

Comment:
"KafkaNodePool is set successfully to 3 replicas" .... it's not accurate or ...
What you see in the `KafkaNodePool` is obviously 3 replicas as you set it but in reality, the operator revert back to 5 in memory because the nodes cannot be scaled down. Not sure if this applies somehow to your testing.